test/unit/src/shardeum/config.test.ts

```diff
@@ -0,0 +1,727 @@
+import fs from 'fs'
+import path from 'path'
+import { FilePaths } from '../../../../src/shardeum/shardeumFlags'
+import { safeStringify } from '@shardeum-foundation/lib-types/build/src/utils/functions/stringify'
+
+// Mock dependencies
+jest.mock('fs')
+jest.mock('path')
+jest.mock('@shardeum-foundation/core', () => ({
+    DevSecurityLevel: {
+        Unauthorized: 0,
+        Low: 1,
+        Medium: 2,
+        High: 3
+    }
+}), { virtual: true })
+
+// Define DevSecurityLevel enum to avoid dependency issues
+enum DevSecurityLevel {
```

Comment:
Once we start having to do this kinda stuff (copy impl code into tests to break up dep chain) - this is a sign that we really ought to extract some code to make it more testable.